timeless/access_control/owner_privileges.py

```diff
@@ -13,6 +13,13 @@
 
 def has_privilege(method=None, resource=None, *args, **kwargs) -> bool:
     """Check if user with Owner role can access a particular resource."""
+    user = flask.g.get("user")
+    if not user:
+        return False
+
+    if getattr(user.role, "name", None) == "owner":
```

Comment:
@vladarefiev You forgot here :)

Reply:
@paulodamaso Excuse me, don't understand. What did I forget?

Reply:
@paulodamaso It looks like correct. If role is `owner` it has privileges for all resources.

Reply:
@vladarefiev Yes, but you should use `RoleType.Owner` here, right?